tests/functional/simple_test.py

```diff
@@ -409,10 +409,8 @@ def test_override_requirements_file(tmpdir):
         'bootstrap-deps=', '-r', 'requirements-bootstrap.txt',
     )
     err = strip_pip_warnings(err)
-    assert err == (
-        'You must give at least one requirement to install '
-        '(see "pip help install")\n'
-    )
+    # pip>=10 doesn't complain about installing an empty requirements file.
+    assert err == ''
```

Comment:
pretty sure this assertion (and the other similar two below) wasn't actually trying to test this specific behavior, it was just asserting the error output as a sanity check during the larger behavior test.